Slim/Formats.pm

```diff
@@ -309,22 +311,31 @@ sub readTags {
 
 			# Bug 14587, sanity check all MusicBrainz ID tags to ensure it is a UUID and nothing more
 			if ( $tag =~ /^MUSICBRAINZ.*ID$/ ) {
+				my @values;
+				my @val;
+				if (ref $value eq 'ARRAY') {
+					@values = @$value;
+				} else {
+					@values = split($prefs->get('splitlist'), $value);
```

Comment:
We have `Slim::Music::Info::splitTag()` to do this job (plus clean up whitespace etc.). Would it be worth using that instead?

https://github.com/LMS-Community/slimserver/blob/public/9.1/Slim/Music/Info.pm#L1005-L1065